README.md

```diff
@@ -44,15 +44,14 @@ docker image rm tabulario/spark-iceberg && docker-compose pull
 ```
 
 ### Use `Dockerfile` In This Repo
-To directly use the Dockerfile in this repo (as opposed to pulling the hosted `tabulario/spark-iceberg` image), change the following line in any of the docker-compose files.
-```diff
--    image: tabulario/spark-iceberg
-+    build: spark/
-```
+
+To directly use the Dockerfile in this repo (as opposed to pulling the pre-build `tabulario/spark-iceberg` image), you can use `docker-compose build`.
```

Comment:
I see that `image:` and `build:` are both included in the docker compose, but if someone wants to use the local Dockerfile, how do they start it up after running `docker compose build`? Wouldn't `docker compose up` still use the `image: tabulario/spark-iceberg`?

Reply:
By default it will pull the image, if you do `docker-compose build`, it will actually build the image and use that one instead (it will actually build that one, and tag it with the name of the image).

Reply:
Also added a section on building 👍🏻